client/extensions/wp-super-cache/components/advanced.jsx

```diff
@@ -0,0 +1,570 @@
+							{
+								components: { strong: <strong /> }
+							}
+						) }
+					</p>
+				</form>
+			</Card>
+		);
+	}
+
+	renderCacheLocation() {
+		const { translate } = this.props;
+		const { cacheLocation } = this.state;
+
+		return (
+			<Card>
+				<form>
+					<FormFieldset>
+						<FormTextInput
+							onChange={ this.handleCacheLocationChange }
+							value={ cacheLocation } />
+						<FormSettingExplanation>
+							{ translate(
+								'Change the location of your cache files. The default is WP_CONTENT_DIR . ' +
+								'/cache/ which translates to {{cacheLocation/}}',
+								{
+									components: {
+										cacheLocation: <span>{ cacheLocation }</span>,
+									}
+								}
+							) }
+						</FormSettingExplanation>
+					</FormFieldset>
+				</form>
+			</Card>
+		);
+	}
+
+	renderCacheTimeout() {
+		const { translate } = this.props;
+		const { cacheTimeout } = this.state;
+		const style = {
+			width: '100px',
+			marginRight: '10px',
+		};
+
+		return (
+			<FormFieldset>
+				<FormLabel htmlFor="wp_max_time">
+					{ translate( 'Cache Timeout' ) }
+				</FormLabel>
+
+				<FormTextInput
+					id="wp_max_time"
+					onChange={ this.handleCacheTimeoutChange }
+					style={ style }
+					type="text"
+					value={ cacheTimeout } />
+				{ translate( 'seconds' ) }
+				<FormSettingExplanation>
+					{
+						translate(
+							'How long should cached pages remain fresh? Set to 0 to disable garbage collection. ' +
+							'A good starting point is 3600 seconds.'
+						)
+					}
+				</FormSettingExplanation>
+			</FormFieldset>
+		);
+	}
+
+	renderScheduler() {
+		const { translate } = this.props;
+		const { clock, timer, type } = this.state.scheduler;
+		const style = {
+			width: '100px',
+			marginLeft: '10px',
+			marginRight: '10px',
+		};
+
+		return (
+			<FormFieldset>
+				<FormLabel>
+					{ translate( 'Scheduler' ) }
+				</FormLabel>
+
+				<FormLabel>
+					<FormRadio
+						checked={ 'interval' === type }
+						onChange={ this.handleSchedulerTypeChange }
+						style= { { marginTop: '12px' } }
+						value="interval" />
+					<span>
+						{ translate( 'Timer' ) }
+						<FormTextInput
+							disabled={ 'interval' !== type }
+							onChange={ this.handleSchedulerTimerChange }
+							style={ style }
+							type="text"
+							value={ timer } />
+						{ translate( 'seconds' ) }
+					</span>
+				</FormLabel>
+				<FormSettingExplanation isIndented>
+					{ translate( 'Check for stale cached files every interval seconds.' ) }
+				</FormSettingExplanation>
+
+				<FormLabel>
+					<FormRadio
+						checked={ 'time' === type }
+						onChange={ this.handleSchedulerTypeChange }
+						style= { { marginTop: '12px' } }
+						value="time" />
+					<span>
+						{ translate( 'Clock' ) }
+						<FormTextInput
+							disabled={ 'time' !== type }
+							onChange={ this.handleSchedulerClockChange }
+							style={ style }
+							type="text"
+							value={ clock } />
+						{ translate( 'HH:MM' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **33** times:
`translate( 'HH:mm', { context: 'momentjs format string (for LT)'} )` *ES Score: 22*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).